daemon/emer-site-id-provider.c

```diff
@@ -34,15 +34,23 @@
 #define LOCATION_CONF_FILE SYSCONFDIR "/metrics/location.conf"
 #define LOCATION_LABEL_GROUP "Label"
 
-static GVariant *
-emer_read_location_label (GKeyFile *kf)
+static void
+emer_read_location_label (GVariantBuilder *builder)
 {
+  g_autoptr (GKeyFile) kf = g_key_file_new ();
+  g_autoptr (GError) err = NULL;
```

Comment:
```suggestion
  g_autoptr(GKeyFile) kf = g_key_file_new ();
  g_autoptr(GError) error = NULL;
```

Conventionally a `GError *` is always called `error`. And conventionally there is no space between `g_autoptr` and `(typename)`. (Yes, that is inconsistent with function calls; no I don't know why.)

Reply:
ok